test-cypress/integration/import/import.server.files.spec.js

```diff
@@ -0,0 +1,137 @@
+describe('Import screen validation', function () {
+    let existingRepo = 'w11' + Date.now();
+    let baseURI = 'http://purl.org/dc/elements/1.1/';
+    let context = 'http://example.org/context';
+    let messageSuccess = 'Imported successfully';
+    let fileToImport = 'italian_public_schools_links.nt.gz';
+    let blankNodeFile = 'bnodes.ttl';
+
+    before(function () {
+        cy.visit('/repository');
+        // Create new one
+        cy.createNewRepo(existingRepo).wait(2000);
+    });
+
+    beforeEach(function () {
+        cy.visit('/import#user').wait(1000);
+        cy.selectRepo(existingRepo);
+    });
+
+    afterEach(function () {
+        cy.checkAllServerFiles(true);
+        cy.resetStatusOfUploadedFiles().wait(1000);
+        cy.checkAllServerFiles(false);
+    });
+
+    after(function () {
+        cy.navigateToPage('Setup', 'Repositories');
+        cy.deleteRepo(existingRepo).wait(1000);
+    });
+
+    it('Test import Server files successfully without changing settings', function () {
+        cy.chooseServerFilesTab();
+        cy.selectDistinctFileToImport(fileToImport);
+        cy.importChangingSettings().wait(2000);
+        cy.verifyMessageOccurs(messageSuccess);
+    });
+
+    it('Test import Server files successfully with changing base uri', function () {
+        cy.chooseServerFilesTab();
+        cy.selectDistinctFileToImport(fileToImport);
+        cy.importChangingSettings(true);
```

Comment:
We will have hard times with these tests with waits. Definitely we need to refactor them.